test/test.js

```diff
@@ -23,8 +23,11 @@ test.cb('rebuild the guetzli binaries', t => {
 		});
 });
 
-test('return path to binary and verify that it is working', async t => {
-	t.true(await binCheck(guetzli, ['--version']));
+test.cb('return path to binary and verify that it is working', t => {
+	binCheck(guetzli, ['--version']).then(result => {
+		t.true(result);
+	});
+	t.end();
 });
```

Comment:
For some reason, the async binCheck isn't being completed before AVA asserts the test. It evaluates to an unresolved Promise, thus `false`. The change to sync lets it actually complete. There are definitely other ways to solve this though.